apps/web/src/components/Layouts/PaginatedListLayout.tsx

```diff
@@ -126,6 +126,7 @@ export const PaginatedListLayout: FC<PaginatedListLayoutProps> = function ({
               <div className="flex items-center justify-start gap-2">
                 Displayed items:
                 <Dropdown
+                  multiple={false}
```

Comment:
This is not necessary. Prop is `false` by default

Reply:
Done! Forgot to remove it after the refactor 👍